GVRf/Sample/gvr-sample/src/org/gearvrf/sample/SampleCubeScript.java

```diff
@@ -41,116 +42,108 @@
 
     private static final float CUBE_WIDTH = 20.0f;
     private GVRContext mGVRContext = null;
-    private GVRSceneObject mFrontBoardObject = null;
-    private GVRSceneObject mFrontBoardObject2 = null;
-    private GVRSceneObject mFrontBoardObject3 = null;
+    private GVRSceneObject mFrontFace = null;
+    private GVRSceneObject mFrontFace2 = null;
+    private GVRSceneObject mFrontFace3 = null;
 
     @Override
     public void onInit(GVRContext gvrContext) {
         mGVRContext = gvrContext;
 
-        GVRTexture front = mGVRContext.loadTexture(new GVRAndroidResource(
-                mGVRContext, R.drawable.front));
-        GVRTexture back = mGVRContext.loadTexture(new GVRAndroidResource(
-                mGVRContext, R.drawable.back));
-        GVRTexture left = mGVRContext.loadTexture(new GVRAndroidResource(
-                mGVRContext, R.drawable.left));
-        GVRTexture right = mGVRContext.loadTexture(new GVRAndroidResource(
-                mGVRContext, R.drawable.right));
-        GVRTexture top = mGVRContext.loadTexture(new GVRAndroidResource(
-                mGVRContext, R.drawable.top));
-        GVRTexture bottom = mGVRContext.loadTexture(new GVRAndroidResource(
-                mGVRContext, R.drawable.bottom));
-
         GVRScene scene = mGVRContext.getNextMainScene();
 
-        mFrontBoardObject = new GVRSceneObject(gvrContext, CUBE_WIDTH,
-                CUBE_WIDTH, front);
-        mFrontBoardObject.setName("front");
-        scene.addSceneObject(mFrontBoardObject);
-        mFrontBoardObject.getTransform().setPosition(0.0f, 0.0f,
-                -CUBE_WIDTH * 0.5f);
-
-        mFrontBoardObject2 = new GVRSceneObject(gvrContext, CUBE_WIDTH,
-                CUBE_WIDTH, front);
-        mFrontBoardObject2.setName("front2");
-        scene.addSceneObject(mFrontBoardObject2);
-        mFrontBoardObject2.getTransform().setPosition(0.0f, 0.0f,
+        FutureWrapper<GVRMesh> future_mesh = new FutureWrapper<GVRMesh>(
```

Comment:
`future_mesh` is a C++-style name - please use `futureMesh', which follows Java conventions.